cmd/plasmacli/cmd/eth/challenge.go

```diff
@@ -14,14 +14,16 @@ import (
 	"strconv"
 )
 
-func init() {
```

Comment:
are you certain this works? I ran into issues where binding flags on the same name with defaults could override a flags input because of the ordering in which they were binded

Reply:
is this about the --trust-node flag? It'll definitely have to be changed here as using this command would override the persistent cli --trust-node flag

Reply:
Tested and my hunch was right:
- pulled this branch
- added print of `fmt.Printf(viper.GetString(gasLimitF))` to first line in challenge.go of runE
- ran plasmacli challenge cmd and `240000` was printed, either `300000` was expected or the `300` since I used `-g 300` on the cmd line.

Here is the fix, don't bind flags until the runE is being ran, otherwise binding by other flags will override the flag used on cmd line